wit/deps/golem-1.2/golem-host-1.2.wit

```diff
@@ -0,0 +1,12 @@
+package golem:api@1.2.0;
```

Comment:
This is not how we should do it (I did a similar thing before and had to undo) because it is not (always) possible to refer to multiple versions of the same package. For example not possible with cargo-component.

On the other hand, there is semantic versioning support in wasmtime so we can do additions to the existing interfaces by increasong the minor version number. (I did not know this when I created `golem:api@1.1.0` next to §golem:api@1.0.0` and because of that we have a lot of code duplication now).

So, do the following:
- increase the version to `1.1.1` in `golem-host.1.1.wit`
- add the new function there